src/cmd/support_archive/tarball_test.go

```diff
@@ -1,13 +1,11 @@
 package support_archive
```

Comment:
since you removing tarball.go, does it make sense to rename this file to archive_test ?

Reply:
sure- just wanted to keep some memories :D